bin/circle-exec

```diff
@@ -13,4 +13,4 @@ trap cleanup EXIT
 circleci config process .circleci/config.yml > $WORK_DIR/config.yml
 cat $WORK_DIR/config.yml
 circleci local execute --config $WORK_DIR/config.yml \
-  --env "CIRCLE_LOCAL=true"
+  --env "CIRCLE_LOCAL=true" "$@"
```

Comment:
What's the @ thing here for? Just wondering.

Reply:
It's for passing the job name to circleci cli, so you can use this script to run the jobs locally. (accidentally removed in a previous pr)